src/Oscoin/Crypto/Address/Bech32.hs

```diff
@@ -0,0 +1,58 @@
+{-- | Partial, opinionated support for Bech32.
+-- Mostly taken and adapted from the reference implementation available
+-- <here https://github.com/sipa/bech32/blob/master/ref/haskell/src/Codec/Binary/Bech32.hs>
+--}
+module Oscoin.Crypto.Address.Bech32
+    ( createChecksum
+    , verifyChecksum
+
+    -- * Interals
```

Comment:
```suggestion
    -- * Internals
```
`:set spell` :wink:

Reply:
You might be surprised by I _do_ code with `:set spell` on. The problem is -- it doesn't persist across buffer changes, so after I while, I forget to re-enable it 😂